src/components/Article.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ article }) => (
+  <div>
+    <h1>{article.title}</h1>
+    <p>{article.text}</p>
+    <span>{article.date}</span>
+  </div>
```

Comment:
```suggestion
  <>
    <h1>{title}</h1>
    <p>{text}</p>
    <span>{date}</span>
  </>
```
Тогда тут можно будет использовать переменные. Так же лучше обернуть в пустой тег, так как твой див будет виден на странице